apps/mobile/src/queries/balance/account-balance.query.ts

```diff
@@ -39,9 +39,9 @@ export function useAccountBalance(accountId: AccountId): AccountBalance {
     sip10AccountBalance.state === 'loading' ||
     runesAccountBalance.state === 'loading';
   const isError =
-    btcAccountBalance.state === 'error' ||
-    stxAccountBalance.state === 'error' ||
-    sip10AccountBalance.state === 'error' ||
+    btcAccountBalance.state === 'error' &&
```

Comment:
I'm not sure about this as if one of these has errored, that means you aren't showing the correct balance then. 

I think it's correct to show an error if any of the balance requests has failed

Reply:
I think we should keep this and focus on erroring less if we can. I'm open to being convinced otherwise though

Reply:
Why not both? We're going to see errors, when we do we should handle them as well as reasonably possible and also optimize for less errors (while stil handling them well)